website/www/site/content/en/documentation/runners/prism.md

```diff
@@ -0,0 +1,103 @@
+Processing Time timers do not yet have real time support.
+
+
+See the [Roadmap](/roadmap/prism-runner/) for how to find current progress.
+Specific feature support information will soon migrate to the [Runner Capability Matrix](/documentation/runners/capability-matrix/).
+
+# Using the Prism Runner
+
+{{< language-switcher go java py >}}
+
+<span class="language-go">Prism is the default runner for the Go SDK and is used automatically. Set the runner with the flag `--runner=PrismRunner`. </span>
+<span class="language-java">Set the runner to `PrismRunner`. </span>
+<span class="language-py">Set the runner to `PrismRunner`. </span>
+
+For other SDKs, Prism is included as an asset on [Beam Github Releases](https://github.com/apache/beam/releases/tag/v{{< param release_latest >}}) for download and stand alone use.
+
+Here are some resources with information about how to test your pipelines.
+<ul>
+  <li><a href="/documentation/pipelines/test-your-pipeline/">Test Your Pipeline</a></li>
+  <li>The <a href="/get-started/wordcount-example/#testing-your-pipeline-with-asserts">Apache Beam WordCount Walkthrough</a> contains an example of logging and testing a pipeline with asserts.
+  <!-- Java specific links -->
+  <li class="language-java"><a href="/blog/2016/10/20/test-stream.html">Testing Unbounded Pipelines in Apache Beam</a> talks about the use of Java classes <a href="https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/index.html?org/apache/beam/sdk/testing/PAssert.html">PAssert</a> and <a href="https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/index.html?org/apache/beam/sdk/testing/TestStream.html">TestStream</a> to test your pipelines.</li>
+</ul>
+
+### Specify your dependency
+
+<span class="language-java">When using Java, you must specify your dependency on the Direct Runner in your `pom.xml`.</span>
+{{< highlight java >}}
+<dependency>
+   <groupId>org.apache.beam</groupId>
+   <artifactId>beam-runners-prism-java</artifactId>
+   <version>{{< param release_latest >}}</version>
+   <scope>runtime</scope>
+</dependency>
+{{< /highlight >}}
+
+<span class="language-py">This section is not applicable to the Beam SDK for Python. Prism is built in.</span>
+<span class="language-go">This section is not applicable to the Beam SDK for Go. Prism is built in.</span>
+
+Except for the Go SDK, Prism is included as an asset on [Beam Github Releases](https://github.com/apache/beam/releases/tag/v{{< param release_latest >}}) for automatic download, startup, and shutdown on SDKs.
+The binary is cached locally for subsequent executions.
+
+## Pipeline options for the Prism Runner
+
+Prism aims to have minimal configuration required, and does not currently present user pipeline options.
+
+## Running Prism Standalone
+
+Prism can be executed as a stand alone binary and will present a basic UI for listing jobs, and job status.
```

Comment:
I might explicitly call out that this requires having go installed on your machine. I think that is reasonable, but people may wonder if the go command below is correct for java/python pipelines